main.go

```diff
@@ -119,3 +130,35 @@ func printVersion() {
 	setupLog.Info(fmt.Sprintf("Git Commit: %s", version.GitCommit))
 	setupLog.Info(fmt.Sprintf("Build Date: %s", version.BuildDate))
 }
+
+// Creates the patterns operator configmap
+// This will include configuration parameters that
+// will allow operator configuration
+func createGitOpsConfigMap() error {
+	// LRC - Declared as global const
+	// operatorConfigFile := "patterns-operator-config"
+	// operatorNamespace := "openshift-operators"
+
+	config, _ := ctrl.GetConfig()
+	clientset, _ := kubernetes.NewForConfig(config)
+
+	configMap := corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ConfigMap",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      controllers.OperatorConfigMap,
+			Namespace: controllers.OperatorNamespace,
+		},
+		Data: controllers.DefaultPatternOperatorConfig,
```

Comment:
We talked about this several times. Default keys must not be written to etcd.

Reply:
Yes we have.  Updated and removed.